autopilot/autopilot_customer_churn_high_level_with_evaluation.ipynb

```diff
@@ -0,0 +1,831 @@
+    "\n",
+    "```python\n",
+    "from sagemaker import AutoML\n",
+    "automl = AutoML.attach(auto_ml_job_name='<autopilot-job-name>')\n",
+    "\n",
+    "test_data = ... # test_data to be used (with target column)\n",
+    "test_file = ... # path of data to upload to S3 and perform batch inference (test_data WITHOUT target column)\n",
+    "target_attribute_name = ... # name of target column (values to predict)\n",
+    "target_attribute_true_value = ... # Second value in target column (binary classification \"True\" class)\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_candidate = automl.describe_auto_ml_job()['BestCandidate']\n",
+    "best_candidate_name = best_candidate['CandidateName']\n",
+    "pprint(best_candidate)\n",
+    "print('\\n')\n",
+    "print(\"CandidateName: \" + best_candidate_name)\n",
+    "print(\"FinalAutoMLJobObjectiveMetricName: \" + best_candidate['FinalAutoMLJobObjectiveMetric']['MetricName'])\n",
+    "print(\"FinalAutoMLJobObjectiveMetricValue: \" + str(best_candidate['FinalAutoMLJobObjectiveMetric']['Value']))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Due to some randomness in the algorithms involved, different runs will provide slightly different results, but accuracy will be around or above $93\\%$, which is a good result."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check Top Candidates\n",
+    "\n",
+    "In addition to the `best_candidate`, we can also explore the other top candidates generated by SageMaker Autopilot. \n",
+    "\n",
+    "We use the `list_candidates` method to see our other top candidates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N # number of top candidates to check, defined at the beginning of the notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "candidates = automl.list_candidates(sort_by='FinalObjectiveMetricValue',\n",
+    "                                    sort_order='Descending',\n",
+    "                                    max_results=N)\n",
+    "\n",
+    "for candidate in candidates:\n",
+    "    print(\"Candidate name: \", candidate['CandidateName'])\n",
+    "    print(\"Objective metric name: \", candidate['FinalAutoMLJobObjectiveMetric']['MetricName'])\n",
+    "    print(\"Objective metric value: \", candidate['FinalAutoMLJobObjectiveMetric']['Value'])\n",
+    "    print('\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Performing Inference <a name=\"Inference\"></a>\n",
```

Comment:
I would call this `Evaluate top candidates`

You can define TOP_N_CANDIDATES (instead of N defined at the top) here. and describe the details here. We may consider parameterizing the instance type used for the transform jobs.

TOP_N_CANDIDATES=5  



`N = 5`

Reply:
Updated the section name and moved the cell to the top of the notebook with new `TOP_N_CANDIDATES` name. Thanks